lib/jekyll/commands/doctor.rb

```diff
@@ -39,20 +39,17 @@ def healthy?(site)
             !conflicting_urls(site),
             !urls_only_differ_by_case(site),
             proper_site_url?(site),
+            properly_gathered_drafts?(site),
             properly_gathered_posts?(site),
           ].all?
         end
 
+        def properly_gathered_drafts?(site)
+          inside_custom_collections_dir?(site, "drafts")
```

Comment:
the directory name should start with an underscore (`_`)
```ruby
inside_custom_collections_dir?(site, "_drafts")
```
or handled appropriately in the private helper method..

Reply:
Ha, right. I’ll modify as soon as I’m back.